### DISCUSSION TITLE
Graphics card issue

### DISCUSSION THREAD

**Discussion by Alex4afk** [end-4/dots-hyprland#314] [Q&A] [answered]
Hi,

i want to talk about an issue i've been having for 5 hours already with no issue. My GPU is a RTX 3060 Ti and i installed all the drivers for it that i could find including multilib. Still, i get wlrbackend null issue. I have attached the crash log file if it helps.I hope i can find a solution ASAP. Thanks!
[crashreport.txt](https://github.com/end-4/dots-hyprland/files/14547209/crashreport.txt)

**Comment by clsty** [comment, marked as answer]
We just have a similar issue https://github.com/end-4/dots-hyprland/issues/312 closed as completed.

Also refer to <https://wiki.hyprland.org/Getting-Started/Master-Tutorial/#launching-hyprland>
